Gemfile.lock

```diff
@@ -111,11 +111,19 @@ GEM
     rb-inotify (0.9.8)
       ffi (>= 0.5.0)
     redcarpet (3.4.0)
+<<<<<<< HEAD
     sass (3.4.24)
     sassc (1.11.2)
       bundler
       ffi (~> 1.9.6)
       sass (>= 3.3.0)
+=======
+    rest-client (2.0.1)
+      http-cookie (>= 1.0.2, < 2.0)
+      mime-types (>= 1.16, < 4.0)
+      netrc (~> 0.8)
+    sass (3.4.25)
+>>>>>>> Switch from sassc to sass gem
```

Comment:
Hot darn! Thanks for catching that. Fixed in 999c3ad06b6000a485b00ebfd35d0cfeac75c7d3.